src/app/static/src/app/index.ts

```diff
@@ -43,7 +38,7 @@ export const router = createRouter({
     routes,
 });
 
-const app = createApp(Hint);
+const app = createApp(Hint, {...(mountEl as HTMLDivElement).dataset});
```

Comment:
Could we do `createApp({components:{Hint}}).mount("#app");` to save having to do the query selector?

Reply:
I might be being a little stupid here but I tried just doing
```JavaScript
const app = createApp({ components: Hint })
```
And I added the hint tag back into the `index.ftl` file. But this did not show anything when you run the app since the Vue app doesn't have a template to render. The query selector is to pass in the data attributes not to mount the app yet. Mounting the app happens at the end of the index file.

Reply:
Yeah, but I was hoping you could avoid that by passing in the values from the template. This approach does work for us in WODIN (see [here](https://github.com/mrc-ide/wodin/blob/main/app/server/views/app.hbs and [here](https://github.com/mrc-ide/wodin/blob/main/app/static/src/app/wodin.ts)), but you're right, it's not doing anything here. I think that may be because WODIN uses a different build target? (`--target lib`)

Reply:
so I just tried this, get a bunch of umd, umd.min, common, etc and when I import the relevant ones into the ftl files it doesn't display anything on the screen, not sure exactly what the problem is here, might be to do with the setup we currently have with hint